js/ws/index.js

```diff
@@ -10,8 +10,9 @@ var wss = new ws.Server({
 
 if (cluster.isWorker) {
   process.on('message', function(msg) {
+    var msgBuf = Buffer(msg);
     wss.clients.forEach(function each(client) {
-      client.send(msg);
+      client.send(msgBuf);
```

Comment:
Ah, yes you saw the same thing I did, but I didn't catch that bit.  The websocket-bench doesn't seem to care about the response being a binary frame, but that's probably something that should fixed there.